utility_scripts/quota_audit.py

```diff
@@ -0,0 +1,171 @@
+# Copyright 2017 Massachusetts Open Cloud
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+"""Audits quota requests
+
+Utility script for finding projects that don't match quota requests.
+
+Optional second flag which specifies path to configuration file. If specified,
+configuration file will be used for authentication. If none specified, the
+script will use environment variables.
+
+Usage:
+    python quota_audit.py [<REQUEST FILE>] [<CONFIG_FILE>]
+"""
+import os
+import csv
+import argparse
+import ConfigParser
+from keystoneclient.v3 import client
+from keystoneauth1 import session
+from keystoneauth1.identity import v2
+from keystoneauth1.exceptions.http import NotFound
+from neutronclient.v2_0 import client as nclient
+from novaclient import client as novaclient
+from cinderclient.v2 import client as cinderclient
+
+
+def diff_moc_quotas(project_quotas):
+
+    different_quotas = {}
+    for resource in moc_standards:
```

Comment:
I should have said this earlier, but maybe, if we're going to have this global, make it `MOC_STANDARDS` to make it clear it is a constant